news/4700.removal

```diff
@@ -0,0 +1,2 @@
+Move all of pip's APIs into the pip._internal package, properly reflecting the
```

Comment:
What happens if there are 2 news files?

Reply:
If the contents are the same, it'll get rolled up into a single news entry, referencing multiple issues.

Reply:
Oh, that's neat. I never realised this was possible.